state/state_test.go

```diff
@@ -3219,7 +3219,11 @@ func (s *StateSuite) TestParseActionTag(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 	f, err := u.AddAction("snapshot", nil)
 	c.Assert(err, jc.ErrorIsNil)
-	action, err := s.State.Action(f.Id())
+
+	model, err := s.State.Model()
+	c.Assert(err, jc.ErrorIsNil)
```

Comment:
As mentioned in the call it might be good to add a `Model` to a base test suite to avoid the need for tests to do this all over the place. `StateSuite` is probably the right place to add it in this case.